### PR TITLE
Chore: Move util methods inside Rusha class

### PR DESCRIPTION
**Pull request by tonyjin**
This fixes Rusha when the class is minified.

### CODE REVIEW

File: rusha.js [hunk 1]
```diff
@@ -1,57 +1,29 @@
...
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
- * IN THE SOFTWARE.
- */
```
Comment (tonyjin) [inline rusha.js]: Not sure why this comment was removed - did it depend on util being defined outside?

**Comment by tonyjin** [comment]
See intended use case: https://github.com/box/box-ui-elements/pull/105/files#diff-54cf1236d7292d039f3284ace0e21badR11

This doesn't work when util is defined outside of Rusha since Rusha can be minified and you can run into issues like the below minified code, note `r.getDataType(e)`:

```
d = function(e) {
        switch (r.getDataType(e)) {
        case "string":
            return l.bind(e);
        case "array":
        case "buffer":
            return u.bind(e);
        case "arraybuffer":
            return u.bind(new Uint8Array(e));
        case "view":
            return u.bind(new Uint8Array(e.buffer,e.byteOffset,e.byteLength));
        case "blob":
            return c.bind(e)
        }
```

File: rusha.js (hunk 1 above)
Comment (srijs) [reply rusha.js]: Yeah, this one is weird. In any case, I'm okay with not having the comment.

**Comment by srijs** [comment]
Good stuff, thanks!

**Comment by srijs** [comment]
Published as 0.8.7.

**Comment by tonyjin** [comment]
Thanks for your work on this! :) We're using this @Box to hash files for chunked uploads (verifying integrity of uploaded chunks and the final reconstructed file).

**Pull request merged**